bigbluebutton-html5/imports/ui/components/audio/audio-modal/container.jsx

```diff
@@ -91,14 +96,15 @@ export default lockContextContainer(withModalMounter(withTracker(({ mountModal,
     outputDeviceId: Service.outputDeviceId(),
     showPermissionsOvelay: Service.isWaitingPermissions(),
     listenOnlyMode,
+    autoJoin,
+    useEchoTest,
     skipCheck,
+    skipCheckOnJoin,
     formattedDialNum,
     formattedTelVoice,
     combinedDialInNum,
     audioLocked: userLocks.userMic,
-    joinFullAudioImmediately: !listenOnlyMode && skipCheck,
-    joinFullAudioEchoTest: !listenOnlyMode && !skipCheck,
-    forceListenOnlyAttendee: listenOnlyMode && forceListenOnly && !Service.isUserModerator(),
+    forceListenOnlyAttendee: forceListenOnly && !Service.isUserModerator(),
```

Comment:
I removed it because the property is only responsible for displaying/hiding the ListenOnly button in the audio modal. Not sure why it was being used to determine how to auto join audio.